src/System.IO.FileSystem/src/System/IO/FileOptions.cs

```diff
@@ -19,18 +21,25 @@ public enum FileOptions
     {
         // NOTE: any change to FileOptions enum needs to be 
         // matched in the FileStream ctor for error validation
+        /// <summary>Indicates that no additional options should be used when creating a <see cref="T:System.IO.FileStream" /> object.</summary>
         None = 0,
+        /// <summary>Indicates that the system should write through any intermediate cache and go directly to disk.</summary>
         WriteThrough = unchecked((int)0x80000000),
+        /// <summary>Indicates that a file can be used for asynchronous reading and writing. </summary>
         Asynchronous = unchecked((int)0x40000000), // FILE_FLAG_OVERLAPPED
-        // NoBuffering = 0x20000000,
+                                                   // NoBuffering = 0x20000000,
```

Comment:
Roslyn formatter moved this commented-out enum value to line up with another comment, likely thinking it was just a comment.